swap_meet/vendor.py

```diff
@@ -1,2 +1,49 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
```

Comment:
If the item is not in the inventory then we want to return false
```suggestion
    def remove(self, item):
        if item in self.inventory:
            self.inventory.remove(item)
            return item
        else:
            return false
        
```